mrblib/haconiwa/runner.rb

```diff
@@ -39,9 +39,12 @@ def waitall(&how_you_run)
     end
 
     def run(options, init_command)
-      if File.exist? @base.container_pid_file
-        Logger.exception "PID file #{@base.container_pid_file} exists. You may be creating the container with existing name #{@base.name}!"
+      begin
+        confirm_existence_pid_file(@base.container_pid_file)
+      rescue
+        Logger.exception $!
```

Comment:
Very sorry to say that I like `rescue => e` much better than using `$.*` variables...